.github/workflows/build.yml

```diff
@@ -27,6 +27,6 @@ jobs:
       - name: Build
         run: cargo build
       - name: Run clippy and fail if any warnings
-        run: cargo clippy -- -D warnings
+        run: cargo clippy -- -W clippy::all -W clippy::nursery -D warnings
```

Comment:
It's ok to specify `-W clippy::all` , but I don't think we need to have `clippy::nursery` in CI yet.